src/core/FilePath.cpp

```diff
@@ -211,6 +211,9 @@ FilePath::FilePath()
     else if (testSetDir(appDirPath + "/share")) {
     }
 #endif
+    // Last ditch test when running in the build directory (mainly for travis tests)
+    else if (testSetDir(QString(KEEPASSX_SOURCE_DIR) + "/share")) {
```

Comment:
If you have KeePassXC already (correctly) installed on your system you don't need to re-run `make install`. Travis on the other hand make a new environment for every run so you have to add a `make install` for building the Release tests.

Alternatively you can direcly remove the QT_DEBUG part and leave the last elseif part so you don't duplicate things (but this doesn't even make sense in the first place as a workaround if things are done in the right way)